src/components/case_manager/MyParas.tsx

```diff
@@ -41,24 +40,68 @@ const MyParas = () => {
     return <div>Loading...</div>;
   }
 
-  return (
-    <div>
-      <PersonCreationForm title={"Add a Para"} onSubmit={handleSubmit} />
+  const headCells: HeadCell[] = [
+    {
+      id: "first_name",
+      label: "First Name",
+      hasInput: true,
+    },
+    {
+      id: "last_name",
+      label: "Last Name",
+      hasInput: true,
+    },
+    {
+      id: "email",
+      label: "Email",
+      hasInput: true,
+    },
+    {
+      id: "phone",
```

Comment:
Let's only keep the first name, last name and email columns for now. We can add more columns to this table over time as we start collecting more data.